projects/go-style-guide/src/app/features/ui-kit/components/toast-docs/toast-docs.component.ts

```diff
@@ -88,6 +88,9 @@ export class ToastDocsComponent {
 
   // you can adjust the duration of the toast by passing in the duration param:
   this.toasterService.toastSuccess({ message: 'You clicked the button!' }, 7000);
+
+  // you can also make the toast dismissable by passing in a "dismissable" param to toastSuccess:
```

Comment:
I assume this doesn't just work for `toastSuccess`, but anything where we're using the `ToastInterface` right? Should we make this documentation a little more general then?
```ts
// you can also make the toast dismissable by setting the "dismissable" property on the ToastInterface param

Reply:
Yep! I wanted it to put it in the interface so that it could be used anywhere. That looks good though, I'll implement that 👍